app/services/TaskService.js

```diff
@@ -115,25 +116,19 @@ var TaskService = function (Task, FileService, UserService, SocketService) {
     };
 
     this.getEstimatedChildren = function (task, next) {
-        self.findVelocity(task, function (err, velocity) {
```

Comment:
it generates bugs with velocity. not accepted tasks don't change their estimates